ui/app/components/namespace-picker.ts

```diff
@@ -163,9 +163,12 @@ export default class NamespacePicker extends Component {
     element.style.display = 'none';
 
     let maxWidth = 240; // Default minimum width
-    const namespaceLinks = document.querySelectorAll('[data-test-namespace-link]');
-    namespaceLinks.forEach((checkmark: Element) => {
-      const width = (checkmark as HTMLElement).offsetWidth;
+    // Calculate the maximum width of the visible namespace options
+    // The namespace is displayed as an HDS::checkmark button, so we need to find the width of the checkmark element
+    this.visibleNamespaceOptions.forEach((namespace: NamespaceOption) => {
```

Comment:
@beagins this one can you smoke test as you're most familiar with how this should work? I smoke tested what I believed was the functionality, but just to double check.

Reply:
This logic was added to handle very long namespace names, looks good to me 👍 
![Screenshot 2025-06-19 at 3 41 16 PM](https://github.com/user-attachments/assets/dd9b7a70-95f3-4dff-8daf-d528c6333366)